src/components/Reown/modals/SendTransactionModal.js

```diff
@@ -0,0 +1,198 @@
+                  onClick={() => navigator.clipboard.writeText(input.txId)}
+                >
+                  <i className="fa fa-copy"></i>
+                </button>
+              )}
+            </div>
+            <div className="text-monospace mt-2">
+              {input?.address}
+              {input?.address && (
+                <CopyButton text={input.address} />
+              )}
+            </div>
+          </div>
+        ))}
+      </div>
+    );
+  };
+
+  const renderOutputs = () => {
+    if (!data?.data?.outputs || data.data.outputs.length === 0) {
+      return null;
+    }
+
+    return (
+      <div className="mb-4">
+        <h6 className="mb-3">{t`Outputs`}</h6>
+        {data.data.outputs.map((output, index) => (
+          <div key={index} className="p-3 bg-light rounded mb-2">
+            <div className="d-flex justify-content-between align-items-start mb-2">
+              <div>
+                <strong>{t`Output ${index + 1}`}</strong>
+              </div>
+              <div>
+                {formatValue(output?.value)} {getTokenSymbol(output?.token)}
+              </div>
+            </div>
+            <div className="text-monospace">
+              {output?.address}
+              <CopyButton text={output?.address} />
+            </div>
+            {output?.data && (
+              <>
+                <div className="mt-2 mb-1"><small>{t`Data field:`}</small></div>
+                <div className="text-monospace">
+                  {output.data.join(',')}
+                </div>
+              </>
+            )}
+          </div>
+        ))}
+      </div>
+    );
+  };
+
+  const handleAccept = () => {
+    onAccept(data?.data);
+  };
+
+  const handleReject = () => {
+    onReject();
+  };
+
+  return (
+    <>
+      <div className="modal-header border-0">
+        <h5 className="modal-title">{t`NEW TRANSACTION`}</h5>
+        <button type="button" className="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div className="modal-body">
+        <div className="mb-4">
+          <h6 className="mb-1">{data?.dapp?.proposer}</h6>
+          <small className="text-muted">{data?.dapp?.url}</small>
+        </div>
+
+        {renderInputs()}
+        {renderOutputs()}
+
+        {data?.data?.changeAddress && (
+          <div>
+            <h6 className="mb-3">{t`Change Address`}</h6>
+            <div className="p-3 bg-light rounded">
+              <div className="text-monospace">
+                {data.data.changeAddress}
+                {data.data.changeAddress && (
+                  <button 
+                    className="btn btn-link btn-sm p-0 ml-2" 
+                    onClick={() => navigator.clipboard.writeText(data.data.changeAddress)}
+                  >
+                    <i className="fa fa-copy"></i>
+                  </button>
+                )}
```

Comment:
Why don't you use the `CopyButton` component?

Reply:
Refactored! Thanks